djangocms_versioning/admin.py

```diff
@@ -826,7 +826,9 @@ def has_add_permission(self, request):
     def has_change_permission(self, request, obj=None):
         """Return True for changelist and False for change view.
         """
-        return obj is None
+        if obj is not None:
```

Comment:
Don't see a test for this change and yet, assuming the above comment is True, it did change the permission logic.